### PR TITLE
Add some pebble checks

### PR DESCRIPTION
**Pull request by nrobinaubertin**
### Overview

Add some pebble checks for the discourse service.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by amandahla — DISMISSED** [review]
LGTM

File: src/charm.py [hunk 1]
```diff
@@ -389,9 +389,22 @@ def _create_layer_config(self) -> ops.pebble.LayerDict:
...
+                "discourse-ready": {
                     "override": "replace",
-                    "http": {"url": f"http://localhost:{SERVICE_PORT}"},
+                    "level": "ready",
+                    "http": {"url": f"http://localhost:{SERVICE_PORT}/srv/status"},
+                },
+                "discourse-alive": {
+                    "override": "replace",
+                    "level": "alive",
+                    "tcp": {"port": SERVICE_PORT, "host": "localhost"},
```
Comment (mthaddon) [inline src/charm.py:400]: I think we need to be very careful about adding any checks that are "alive" as I believe that means if they fail the container will be restarted. Are we sure that's what we want here? Will this be responding correctly during asset compilation or s3 uploads, for instance, or will this mean the container gets restarted mid-way through?

Comment (nrobinaubertin) [reply src/charm.py:400]: From what I understand, this check will only have an effect after the 'ready' checks pass. That means that the 'ready' http check will be valid when this check kicks in (which already mean that the unicorn process has started).  
Failing this check doesn't mean that the container will be restarted from what I understand, but only the service ?

Comment (nrobinaubertin) [reply src/charm.py:400]: After testing locally, I can reproduce the error of the CI, only when the alive check is there. So the issue is definitely coming from there.

Comment (nrobinaubertin) [reply src/charm.py]: I removed the check

**Review by amandahla — APPROVED** [review]

**Review by gtrkiller — APPROVED** [review]
LGTM

**Pull request merged**